doc/_extensions/zephyr/manifest_projects_table.py

```diff
@@ -0,0 +1,141 @@
+"""
+Manifest Revisions Table
+========================
+
+This extension allows to render a table containing the revisions of the projects
+present in a manifest file.
+
+Usage
+*****
+
+This extension introduces a new directive: ``manifest-projects-table``. It can
+be used in the code as::
+
+    .. manifest-projects-table::
+        :filter: active
+
+where the ``:filter:`` option can have the following values: active, inactive, all.
+
+Options
+*******
+
+- ``manifest_projects_table_manifest``: Path to the manifest file.
+
+Copyright (c) Nordic Semiconductor ASA 2022
```

Comment:
To keep divergence somewhat under control can you please add the "upstream" URL where this came from?

Reply:
There is no intention to keep this compatible or in sync with "upstream"